lib/osf-components/addon/components/contributor-list/template.hbs

```diff
@@ -1,11 +1,19 @@
-{{#inline-list items=contributorList truncate=3 as |l|}}
-    {{#if l.item}}
-        {{~contributor-list/contributor useLink=useLink contributor=l.item~}}
-    {{else if l.truncate}}
-        {{#if useLink}}
-            <a href='/{{nodeId}}'>{{t 'contributor_list.and_x_more' x=rest}}</a>
-        {{else}}
-            {{~t 'contributor_list.and_x_more' x=rest}}
+{{#if truncated}}
+    {{#inline-list items=contributorList truncate=3 as |l|}}
+        {{#if l.item}}
+            {{~contributor-list/contributor useLink=useLink contributor=l.item~}}
+        {{else if l.truncate}}
+            {{#if useLink}}
+                <a href='/{{nodeId}}'>{{t 'contributor_list.and_x_more' x=rest}}</a>
+            {{else}}
+                {{~t 'contributor_list.and_x_more' x=rest}}
+            {{/if}}
         {{/if}}
-    {{/if}}
-{{/inline-list}}
+    {{/inline-list}}
+{{else}}
+    {{#inline-list items=contributorList as |l|}}
+        {{#if l.item}}
+            {{~contributor-list/contributor useLink=useLink contributor=l.item~}}
+        {{/if}}
+    {{/inline-list}}
+{{/if}}
```

Comment:
I think a better approach would be to make a computed for `truncate` that returns `3` when `truncated` is `true` and `undefined` when `truncated` is `false`. Then just have:
```hbs
{{#inline-list items=contributorList truncate=this.truncate as |l|}}
    {{#if l.item}}
        {{~contributor-list/contributor useLink=useLink contributor=l.item~}}
    {{else if l.truncate}}
        {{#if useLink}}
            <a href='/{{nodeId}}'>{{t 'contributor_list.and_x_more' x=rest}}</a>
        {{else}}
            {{~t 'contributor_list.and_x_more' x=rest}}
    {{/if}}
{{/inline-list}}
```